### DISCUSSION TITLE
Multiple NOTIFICATION_TELEGRAM_CHAT_ID

### DISCUSSION THREAD

**Discussion by danielehrhardt** [upptime/upptime#674] [Q&A] [answered]
Is it possible to send Telegram Notifications to multiple Persons?

**Comment by AnandChowdhary** [comment]
Unfortunately this is currently not supported.

**Comment by Gonchito** [comment, marked as answer]
you can create a telegram group and copy the group id and it works, I configured it like this.

**Reply by danielehrhardt** [reply]
good idea :) thank you
